### ISSUE TITLE
Missing schematic

### ISSUE THREAD

**Issue by berndroid** [EVNotify/EVNotiPi#56]
First of all thank you for this great project.

The software is reasonably well documented, but unfortunately a schematic is missing to assemble the necessary hardware. In the list of necessary hardware there is e.g. a circuit board linked (i2c watchdog and power supply). The corresponding Adafruit Trinket M0 is not linked. What is the Trinket for? A documentation how to link the Trinket is missing completely. Could you please improve it here and publish a complete schematic?

**Comment by GPlay97** [comment]
Personally I'm not using the Trinket version.

Maybe @noradtux can help you better with this

**Comment by berndroid** [comment]
There are different options? The more important would be a schematic for each version. As already said, a hardware documentation would be super important. I would be happy to help you with that if I knew how everything fits together.

**Comment by noradtux** [comment]
Schematics are here:
https://github.com/noradtux/evnotipi-watchdog/tree/master/hardware/EVNotipi-Watchdog_Trinket_M0
The board is just a carrier for the power switch, power supply and voltage scaling.

**Comment by berndroid** [comment]
Thanks for the link. But I meant a schematic for the whole EVNotiPi with all its components. 
You should aggregate all components on one page instead of distributing single pages all over Github. Just my 5 Cents.

**Comment by GPlay97** [comment]
But all hardware and wiring is documented? Or am I missing something? I used the current documentation recently for building the new prototypes

**Comment by berndroid** [comment]
Either I'm too stupid for github or you hid the schematic too well ;)
I can't find a schematic here on the site.

**Comment by GPlay97** [comment]
The readme explains the schematic and required hardware and wiring

**Comment by berndroid** [comment]
The readme unfortunately does not explain how e.g. noratux' version is wired with Trinket or what the advantage of this version is. In my opinion, the readme does not explain how EVERYTHING is wired together.  Only the connection between OBD2 and the corresponding shield is shown. The prototype on Twitter also shows a D-SUB interface and two more boards. One of them is a voltage regulator, which is not in the list of required hardware. Then a second board with two thick capacitors is shown. What is this one for? It is also not in the list. At some point there was a version with a WeMos chip with a relay. Which is the current state of the hardware? I'm confused. I hope you understand what I mean. Your project is so brilliant that I would like to build it, but I don't know how to connect the components.

**Comment by GPlay97** [comment]
I see your point. You are right. Since we also currently experimenting what the best combination is.

The wemos one was not as good as expected and was replaced with the one you saw on Twitter. @noradtux Trinket is an even advanced one.

I think once the prototype is really stable, I can extend the documentation more

**Comment by berndroid** [comment]
Sounds good ;)

Can you tell me what kind of circuit board you can see in the photo on Twitter in the left corner below the green terminals? Or is that still top secret? ;)

**Comment by GPlay97** [comment]
That is the WatchDog, something like the one from @noradtux

**Comment by noradtux** [comment]
Yeah, currently those separate components pretty much are separate projects.

**Comment by GPlay97** [comment]
This has also legal reasons, since I have to only use one hardware combination. And since I build the prototypes currently, it's actually not fixed yet. But once it is, we can extend the documentation later
